test/CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ target_link_libraries(test__expression PUBLIC test_main)
 add_test(NAME Expression
         COMMAND test__expression)
 
-############################## Expression
+############################## Parser
```

Comment:
hahah, some copy-paste issues over here :wink:

Reply:
Such things happen in copy-paste-driven development :smile:.